### PR TITLE
Fix/coverity issue 210406

### PR DESCRIPTION
**Pull request by lhs8928**
Commit1: Handle Unchecked return value issue

Added if statement to check return value

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

Commit2: Handle Uninitialized scalar field issue

Added default constructor to initialize scalar field

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

Commit3: Handle Missing break in switch issue

Added break

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

File: test/unittest/unittest_nntrainer_graph.cpp [hunk 1]
```diff
@@ -27,6 +27,10 @@ class nntrainerGraphTest
       std::tuple<const char *, const IniTestWrapper::Sections, int>> {
 
 protected:
+  nntrainerGraphTest(){
```
Comment (zhoonit) [inline test/unittest/unittest_nntrainer_graph.cpp:30]: Not critical but standard way is to use
```cpp
nntrainerGraphTest(): failAt(0), name("") {};
```

File: test/unittest/unittest_nntrainer_models.cpp [hunk 2]
```diff
@@ -457,6 +457,10 @@ class nntrainerModelTest
       const IniTestWrapper, const nntrainer::TensorDim, const unsigned int>> {
 
 protected:
+  nntrainerModelTest() {
```
Comment (zhoonit) [inline test/unittest/unittest_nntrainer_models.cpp:460]: Same here

File: test/include/nntrainer_test_util.h [hunk 3]
```diff
@@ -213,7 +213,11 @@ class IniTestWrapper {
    * @brief erase ini
    *
    */
-  void erase_ini() noexcept { remove(getIniName().c_str()); }
+  void erase_ini() noexcept {
+    if (remove(getIniName().c_str())) {
+      std::cerr << "failed while remove file\n";
```
Comment (dongju-chae) [inline test/include/nntrainer_test_util.h:218]: Typo?

Comment (dongju-chae) [reply test/include/nntrainer_test_util.h:218]: Also, how about printing `errno` as well.

File: test/unittest/unittest_nntrainer_graph.cpp (hunk 1 above)
Comment (lhs8928) [reply test/unittest/unittest_nntrainer_graph.cpp]: Done

File: test/unittest/unittest_nntrainer_models.cpp (hunk 2 above)
Comment (lhs8928) [reply test/unittest/unittest_nntrainer_models.cpp]: Done

File: test/include/nntrainer_test_util.h (hunk 3 above)
Comment (lhs8928) [reply test/include/nntrainer_test_util.h]: Done

File: test/include/nntrainer_test_util.h [hunk 4]
```diff
@@ -213,7 +213,12 @@ class IniTestWrapper {
    * @brief erase ini
    *
    */
-  void erase_ini() noexcept { remove(getIniName().c_str()); }
+  void erase_ini() noexcept {
+    int status = remove(getIniName().c_str());
+    if (status) {
+      std::cerr << "Failed while removing file: errno is " << status << "\n";
```
Comment (dongju-chae) [inline test/include/nntrainer_test_util.h:219]: You should use `errno` not `status`. Please refer to`man remove`.

**Review by zhoonit — APPROVED** [review]
LGTM after all the concerns are resolved :)

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LTGM

**Review by kimjh12 — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**